libcudacxx/include/cuda/std/__bit/reference.h

```diff
@@ -22,27 +20,36 @@
 #  pragma system_header
 #endif // no system header
 
-  _LIBCUDACXX_PUSH_MACROS
-#include <__undef_macros>
+#include <cuda/std/__algorithm/copy_n.h>
+#include <cuda/std/__algorithm/fill_n.h>
+#include <cuda/std/__algorithm/min.h>
+// TODO: modularize bit a bit
+#include <cuda/std/bit>
+// #include <cuda/std/__bit/countr.h>
+// #include <cuda/std/__bit/invert_if.h>
+// #include <cuda/std/__bit/popcount.h>
+#include <cuda/std/__iterator/iterator_traits.h>
+#include <cuda/std/__memory/construct_at.h>
+#include <cuda/std/__memory/pointer_traits.h>
+#include <cuda/std/__type_traits/conditional.h>
+#include <cuda/std/__utility/swap.h>
+#include <cuda/std/detail/libcxx/include/cstring>
```

Comment:
Question: We want to get rid of `detail/libcxx/include` at some point. Can we move this into e.g
`cuda/std/__string/cstring.h` ?

Reply:
Eh, I think I'd rather do this in a "remove detail/libcxx/include" PR.